organization/planning/plan-of-work-2021.md

```diff
@@ -0,0 +1,225 @@
+* Reduced data formats (NanoAOD/DAOD_PHYSLITE)
+  * Attempted conversations with the non-mainstream analyses
+  * Should we revisit questions of policy (i.e. how to engage user base), consequences (framework design, sites)
+  * Also haven’t spoken to WLCG/DOMA in a year (<https://indico.cern.ch/event/890991/manage/timetable/#20200323>), try to reconnect? But needed more concreteness on what studies should be done.
+
+Besides these, we discussed commonalities with Training WG. One question: how much should HSF branch into (experiment-agnostic) analysis tutorials?
+
+* Curriculum has a basic CMS example — without going heavily in on teaching physics, could this be expanded to cover statistical tools, demonstrate analysis ecosystem e.g. PyHEP? But don’t want to be prescriptive.
+* (Probably) have had discussions comparing e.g. CMS DA school and ATLAS SW tutorial? Can HSF material usefully be integrated into these?
+
+### Detector Simulation
+
+* Topics and cross-group areas:
+  * Simulation for liquid noble gas detectors (covering both neutrino and dark matter experiments)
+  * Use of FLUKA/MARS by experiments
+  * Geometry: e.g. developments/experiences with DD4hep, VecGeom
+  * Dealing with pile-up (possible cross-link with Frameworks, Reconstruction?)
+  * Computational challenges in future detectors (e.g. muon colliders)
+  * Use of GPUs and similar accelerators
+  * Developments in ML for simulation
+* Will keep in touch with Geant4 Task Force for R&D to avoid duplication and identify possible joint meetings
+* Further ideas and input very welcome, especially for cross-WG items, e.g.
+  * Use of Python with/for Simulation?
+  * Training: link up with Geant4 material/courses?
+
+### Reconstruction and Software Triggers
+
+* Areas of common interest (in general these are topics related to HL-LHC and off-LHC experiments, LHC Run 3 R&D is basically closed):
+  * ML for and its integration into production code
+    * Machine learning frameworks for performant inference engines in real-time supporting various hardware backends (possible links to fastML, clariphy, ...)
+    * make the step from conceptual studies to production code integration
+  * Alignment / calibration 
+    * points of collaboration on "sharp knife" tools
+  * Particle flow
+    * was initially planned in last term but did not materialize
+  * Event data formats and geometry (overlaps with other groups)
+    * great possibility to have an overlap session with simulation
+  * Use of heterogeneous architectures
+    * establish small scale demonstrators outside the experiments' structure
+    * investigate inclusion of common HEP reconstruction & trigger packages into such systems
+
+* Establish closer link to training group, expanding from pure technical training
+  *  e.g.: general introduction to HEP triggers and reconstruction and introduction to the methods used; can training be done on little demonstrators:
+    *  ALLEN demonstrator of mixing CPU/GPU algorithm
```

Comment:
Suggest instead: Allen demonstrator of multi-event scheduling framework for GPUs, allowing to interleave CPU and GPU algorithms, and to compile
```suggestion
    *  ALLEN demonstrator of multi-event scheduling framework for GPUs, allowing to interleave host(CPU) and device(CPU / AMD GPU / Nvidia GPU) algorithms, where the device type is chosen at compile time (same source code for all device types)
```